lib/pact/cap.ex

```diff
@@ -0,0 +1,14 @@
+defmodule Kadena.Pact.Cap do
```

Comment:
@santiagoboterokommit why not use `Types.Cap`, it makes no sense to me to implement a behavior to build a single alias of `Cap`.